mrp_bom_current_stock/wizard/bom_route_current_stock.py

```diff
@@ -132,15 +129,20 @@ class BomRouteCurrentStockLine(models.TransientModel):
     location_id = fields.Many2one(
         comodel_name="stock.location",
         string="Source location",
+        redonly=True
```

Comment:
this should be editable, the user should have the possiblity to change the location to look for stock in every line.